org.purei.Play.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: org.purei.Play
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: Play
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
+  - --device=all
+  - --socket=pulseaudio
+modules:
+  - "deps/Dependencies/shared-modules/glu/glu-9.json"
+  - "deps/Dependencies/shared-modules/glew/glew.json"
+  - name: openal
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DALSOFT_REQUIRE_PULSEAUDIO=ON
+      - -DALSOFT_BACKEND_PIPEWIRE=off
+    cleanup: 
+        - /bin
+        - /include
+        - /lib/pkgconfig
+        - /share
+    sources:
+      - type: git
+        url: https://github.com/kcat/openal-soft.git
```

Comment:
you need a specific commit/tag

Reply:
Is it needed @jpd002? I haven't tested much, it did compile on my end without.
I suppose either it is optional or the project build it on its own.

Reply:
the project doesnt build this no, thought to my surprise just built this locally and it worked with out (aka its part of the kde sdk `Found OpenAL: /usr/lib/x86_64-linux-gnu/libopenal.so`)